scripts/babel-relay-plugin/src/GraphQLPrinter.js

```diff
@@ -290,6 +290,14 @@ function printFieldsAndFragments(
         fragments.push(printQueryFragment(selection, options));
       } else if (selection.kind === kinds.FIELD) {
         fields.push(selection);
+
+        if (
+          isConnectionType(type) &&
```

Comment:
`!!` is a common pattern for "cast to boolean". It's nice to have boolean expressions in conditionals, but not required.

```javascript
null && true // null
!!null && true // false